src/assessor_tools.yml

```
@@ -226,6 +240,14 @@
         python2: yes
         unarchive_extra_opts:
           - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/Hackndo/lsassy/tarball/master
+        install_dir: /tools/lsassy
+        pip_packages:
+          - lsassy
```

If you're installing `lsassy` from source, then why are you also installing it as a `pip` package?  Note also that the `Hackndo/lsassy` repo has a `setup.py` file in its root:
```suggestion
          - .
```

---

I wasn't familiar with the proper way to install requirements when a `requirement.txt` file wasn't present. Now I know and will do so in the future.